app/views/AssociatedCompaniesView.scala.html

```diff
@@ -34,6 +34,7 @@
 
 @(form: Form[_], associatedCompaniesParameter: AssociatedCompaniesParameter,mode: Mode)(implicit request: Request[_], messages: Messages)
 
+@heading= {@AssociatedCompaniesViewHelper.heading(associatedCompaniesParameter)}
```

Comment:
if we use different syntax we can stop this being coerced to Html 
```suggestion
@heading= @{AssociatedCompaniesViewHelper.heading(associatedCompaniesParameter)}
```

Reply:
and that will mean we can pass Text() to content, like `legend = LegendViewModel(content = Text(heading))` instead of using `HtmlContent`